appengine/firebase-tictactoe/README.md

```diff
@@ -0,0 +1,62 @@
+# Tic Tac Toe on Google App Engine Standard using Firebase
+
+This directory contains a project that implements a realtime two-player game of
+Tic Tac Toe on Google [App Engine Standard][standard], using the [Firebase] database
+for realtime notifications when the board changes.
+
+[Firebase]: https://firebase.google.com
+[standard]: https://cloud.google.com/appengine/docs/about-the-standard-environment
+
+## Prerequisites
+
+* Install [Apache Maven][maven] 3.0.5 or later
+* Create a project in the [Firebase Console][fb-console]
+* Install the [Google Cloud SDK][sdk]
+* For staging locally, you must supply credentials that would otherwise be
+  inferred from the App Engine environment. Download [service account
+  credentials][creds] and set the `GOOGLE_APPLICATION_CREDENTIALS` environment
+  variable to its path:
+
+        export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your/credentials.json
+
+* In the [Overview section][fb-overview] of the Firebase console, click 'Add
+  Firebase to your web app' and replace the contents of the file
+  `src/main/webapp/WEB-INF/view/firebase_config.jspf` with that code snippet.
+* Edit
+  [`src/main/webapp/WEB-INF/appengine-web.xml`](src/main/webapp/WEB-INF/appengine-web.xml)
```

Comment:
Really isn't required anymore, I typically use the string `no-longer-required`, the next Java SDK release will let us delete the line.